### PR TITLE
Get-DbaService - Skip after first success / added more verbose

### LINKED ISSUE #7033: [Bug] Get-DbaService - No services found in relevant namespaces
**Report**

Host used
[ X] powershell.exe
[X ] ISE
 VS Code
 Other (please specify)

Errors Received
Transcript started, output file is C:\Users\SOMEUSER\Documents\PowerShell_transcript.DESKTOP-4GBJ93T.YojLPlol.20201214111123.txt
PS C:\Users\SOMEUSER\OneDrive\Documents\Clients\PowerShell\dbatools-master> Get-DbaService -ComputerName $env:COMPUTERNAME
PS C:\Users\SOMEUSER\OneDrive\Documents\Clients\PowerShell\dbatools-master> TerminatingError(): "Multiple protocol connections were attempted, but no successful connections could be established with the specified computer."
>> TerminatingError(): "Multiple protocol connections were attempted, but no successful connections could be established with the specified computer."
PS C:\Users\SOMEUSER\OneDrive\Documents\Clients\PowerShell\dbatools-master> TerminatingError(): "Multiple protocol connections were attempted, but no successful connections could be established with the specified computer."
>> TerminatingError(): "Multiple protocol connections were attempted, but no successful connections could be established with the specified computer."
PS C:\Users\SOMEUSER\OneDrive\Documents\Clients\PowerShell\dbatools-master> TerminatingError(): "Multiple protocol connections were attempted, but no successful connections could be established with the specified computer."
>> TerminatingError(): "Multiple protocol connections were attempted, but no successful connections could be established with the specified computer."
WARNING: [11:11:31][Get-DbaService] No services found in relevant namespaces on DESKTOP-4GBJ93T. Please note that this function is available from SQL 2005 up.
PS C:\Users\SOMEUSER\OneDrive\Documents\Clients\PowerShell\dbatools-master> Stop-Transcript

Steps to Reproduce
   1. Get-DbaService $env:ComputerName on a new PS session 
   2. Repeat same command now returns zero results and warning 

Expected Behavior
Services as per initial execution 

Actual Behavior
WARNING: [10:41:43][Get-DbaService] No services found in relevant namespaces on SOMECOMPUTER. Please note that this
function is available from SQL 2005 up.

Environmental information
🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
Please provide the output of the below script

PowerShell Version : 5.1.18362.1171
dbatools latest installed : 1.0.134
Culture of OS : en-AU

SQL Server:

us_english

Modules Active 
Script     1.0.134    dbatools                            {Select-DbaObject, Set-DbatoolsConfig, Add-DbaAgDatabase, ...
Manifest   3.1.0.0    Microsoft.PowerShell.Management     {Add-Computer, Add-Content, Checkpoint-Computer, Clear-Con...
Manifest   3.1.0.0    Microsoft.PowerShell.Utility        {Add-Member, Add-Type, Clear-Variable, Compare-Object...}
Binary     1.0.0.1    PackageManagement                   {Find-Package, Find-PackageProvider, Get-Package, Get-Pack...
Script     1.0.0.1    PowerShellGet                       {Find-Command, Find-DscResource, Find-Module, Find-RoleCap...
Script     2.0.0      PSReadline                          {Get-PSReadLineKeyHandler, Get-PSReadLineOption, Remove-PS...

### PR DESCRIPTION
**Pull request by andreasjordan**
## Type of Change

 - [x] Bug fix (non-breaking change, fixes #7033 )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 
Not sure yet if it really fixes #7033, but it will help to find the root cause with more verbose messages. 
And generaly the command is now faster because it only fetches the services from the highest namespace found.

### CODE REVIEW

File: functions/Get-DbaService.ps1 [hunk 1]
```diff
@@ -240,14 +237,14 @@ function Get-DbaService {
...
 
                             $ClusterServiceTypeList = @(1, 2, 5, 7)
                             if ($ClusterServiceTypeList -contains $service.SQLServiceType) {
-                                Add-Member -Force -InputObject $service -MemberType NoteProperty -Name Clustered -Value ($serviceAdvancedProperties | Where-Object PropertyName -eq 'CLUSTERED' ).PropertyNumValue
-                                Add-Member -Force -InputObject $service -MemberType NoteProperty -Name VSName -Value ($serviceAdvancedProperties | Where-Object PropertyName -eq 'VSNAME' ).PropertyStrValue
+                                Add-Member -Force -InputObject $service -MemberType NoteProperty -Name Clustered -Value ($serviceAdvancedProperties | Where-Object PropertyName -EQ 'CLUSTERED' ).PropertyNumValue
+                                Add-Member -Force -InputObject $service -MemberType NoteProperty -Name VSName -Value ($serviceAdvancedProperties | Where-Object PropertyName -EQ 'VSNAME' ).PropertyStrValue
```
Comment (wsmelton) [inline functions/Get-DbaService.ps1:247]: Please revert the casing changes like this.

**Review by wsmelton — CHANGES_REQUESTED** [review]

File: functions/Get-DbaService.ps1 (hunk 1 above)
Comment (andreasjordan) [reply functions/Get-DbaService.ps1:247]: Oh, I think that was the Invoke-DbatoolsFormatter and I didn't noticed. Thanks.

**Comment by potatoqualitee** [comment]
ohh i was hoping this command could get faster! thank you for that technique.

@nvarscar - can you please give a review as well if you've got time?

**Review by nvarscar — APPROVED** [review]
looks reasonable to me

**Comment by potatoqualitee** [comment]
excellent! thank you both 🥳

**Pull request merged**